ci-operator/step-registry/osd-ccs/cluster/provision/gcp/osd-ccs-cluster-provision-gcp-commands.sh

```diff
@@ -18,25 +106,33 @@ SUBSCRIPTION_TYPE=${SUBSCRIPTION_TYPE:-"standard"}
 REGION=${REGION:-"${LEASED_RESOURCE}"}
 CLUSTER_TIMEOUT=${CLUSTER_TIMEOUT}
 
+if [ -f "${SHARED_DIR}/xpn.json" ]; then
+  IS_XPN=1
```

Comment:
Do we have another value rather than `0 or 1`. If not, I think 'yes' or 'no' could be better. Besides, I'd like we have an environment variable(e.g. `ENABLE_SHARED_VPC`) to control whether we use it or not but not depended on the file.